### ISSUE TITLE
Generating export code generates page conditionals as post IDs instead of slugs

### ISSUE THREAD

**Issue by rask** [elliotcondon/acf#498]
When exporting field groups ACF (PRO?) sets field group location conditional *Page* as post ID, not page slug/title. This results in exported PHP field code being unusable in other environments than where the field group was generated in (as post IDs do not match 99% of the time). I tried to hack and set page title and slug in place of the ID in the generated export code but it had no effect.

I think this was slug based previously? Or has it always been this way?

This happens on the latest ACF PRO at least.

Output in the PHP code looks as follows:

```
        ...
        'location' => array(
            array(
                array(
                    'param' => 'page',
                    'operator' => '==',
                    'value' => '11',
                ),
            ),
        ),
        ...
```

**Comment by elliotcondon** [comment]
Hi @rask 

Thanks for the ticket,

This location rule has always been the post ID, the same as all page/post location rules.

I will consider changing this to the page slug, but I encourage you to create your own location rule to do this in the meantime - very easy
http://www.advancedcustomfields.com/resources/custom-location-rules/

**Comment by rask** [comment]
Then I consider myself lucky as my groups have worked across installations as is. :D

I created a custom rule for slug checking the instant I discovered this one didn't operate as I expected. Was quite easy as you mentioned. :+1:

Perhaps keep the ID check as is (to not break old installations) and introduce something like post/page slug and maybe even page path later on? As I said currently checking against IDs only may break the field groups when transferred between installations. Not a big problem when we're talking about 1 or two fields but when we have dozens that need to be validated it may get time consuming to test each one.
